src/AboutPage/index.js

```diff
@@ -52,6 +52,20 @@ function AboutPage() {
           </Button>
         </NewTabLink>
       </p>
+      <p>
+        <NewTabLink href="mailto:helpcovidmaps@gmail.com">
```

Comment:
can we move these two below the privacy policy?

Reply:
Sounds good, will make that change